docs/swagger/spec.yml

```diff
@@ -35,17 +35,6 @@ paths:
         - $ref: '#/parameters/orderingBase'
         - $ref: '#/parameters/pageSize'
         - $ref: '#/parameters/page'
-        - $ref: '#/parameters/organization'
-        - name: status
-          in: query
-          description: Filter users by active/inactive status
-          required: false
-          type: string
-        - name: role
-          in: query
-          description: Role to filter users by (e.g. get all admin users)
-          required: false
-          type: string
       responses:
         200:
           description: Paginated list of users
```

Comment:
While you're in here: a few lines above here there's a "Get a list of user" that can use some pluralization.